R/pkg/R/window.R

```diff
@@ -17,23 +17,23 @@
 
 # window.R - Utility functions for defining window in DataFrames
 
-#' window.partitionBy
+#' windowPartitionBy
 #'
 #' Creates a WindowSpec with the partitioning defined.
 #'
-#' @rdname window.partitionBy
-#' @name window.partitionBy
+#' @rdname windowPartitionBy
+#' @name windowPartitionBy
 #' @export
 #' @examples
 #' \dontrun{
-#'   ws <- window.partitionBy("key1", "key2")
+#'   ws <- windowPartitionBy("key1", "key2")
 #'   df1 <- select(df, over(lead("value", 1), ws))
 #'
-#'   ws <- window.partitionBy(df$key1, df$key2)
+#'   ws <- windowPartitionBy(df$key1, df$key2)
 #'   df1 <- select(df, over(lead("value", 1), ws))
 #' }
-#' @note window.partitionBy(character) since 2.0.0
-setMethod("window.partitionBy",
+#' @note windowPartitionBy(character) since 2.0.0
+setMethod("windowPartitionBy",
```

Comment:
minor comment: Can we document the parameter `@param col` that is in all the 4 functions ? That'll also remove some of the CRAN warnings